bot/resources/evergreen/py_topics.yaml

```diff
@@ -38,9 +42,10 @@
     - What feature would you like to see added to the library? what feature in the library do you think is redundant?
     - Do you think there's a way in which Discord could handle bots better?
 
-# esoteric-python
+# esoteric-Python
```

Comment:
Is there any reason for captializing Python here? The text channel names are all lowercase in the guild.

Reply:
I did a replace-all for "python" to "Python", and I had forgotten that `#esoteric-python` is a channel that exists.

This has been reverted to `esoteric-python` to keep consistency with the actual channels on the server.

Reply:
`Python` is also capitalized in some of the other channel names as well.

Reply:
Fixed all of them.